### PR TITLE
SW-3626 Storybook Integration

### PR DESCRIPTION
**Pull request by charlie-tf**
- Added storybook using `npx storybook init`
- Correctly initialize theme in preview.tsx
- Add a story for the TfMain component to prove things are working

image

### CODE REVIEW

**Comment by charlie-tf** [comment]
Ack, the build is broken for the app now 😭 digging into this

File: .storybook/preview.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        color: /(background|color)$/i,
+        date: /Date$/,
+      },
+    },
+  },
+};
+
+export default preview;
+
+import theme from '../src/theme';
```
Comment (karthikbtf) [inline .storybook/preview.tsx:18]: nice 👍

**Review by karthikbtf — COMMENTED** [review]
Looks great, nice upgrade for us and enables new workflows, thanks!
The build issue sure is strange.. not sure why the node version confusion.

**Review by karthikbtf — APPROVED** [review]
🎉

**Comment by charlie-tf** [comment]
So the webpack dependency addition was causing a different error locally. Taking that out solved local build. However, the Vercel deploy still failed. I updated the node version Vercel is using to 16 and attempting to redeploy...

**Comment by karthikbtf** [comment]
> So the webpack dependency addition was causing a different error locally. Taking that out solved local build. However, the Vercel deploy still failed. I updated the node version Vercel is using to 16 and attempting to redeploy...

Ok 🤞

**Comment by charlie-tf** [comment]
> So the webpack dependency addition was causing a different error locally. Taking that out solved local build. However, the Vercel deploy still failed. I updated the node version Vercel is using to 16 and attempting to redeploy...

And it worked! 🎉

**Pull request merged**